src/main/java/com/tstepnik/planner/service/TaskService.java

```diff
@@ -46,6 +46,9 @@ public Task addTask(Task task) {
         if (task.getImportance() == null) {
             task.setImportance(DEFAULT_IMPORTANCE);
         }
+        if (task.getDone() == null) {
```

Comment:
Do we really need it here? I am not sure because this code has been also changed on different branch

Reply:
I removed it,probably in the last commits, you can missed it and had been looking at that one before this change.